stable/README.md

```diff
@@ -1,167 +1,167 @@
-# Getting Started with NuoDB/Helm on OpenShift
+### The instructions on this page are in two parts:
 
-This post will walk you through getting both the Tiller server and Helm client up and running on OpenShift, and then installing your first NuoDB Helm Chart. It assumes that you already have the OpenShift `oc client` installed locally and that you are logged into your OpenShift instance.
+1. **[Getting Started with Helm][4]** covers how to install and configure Helm on a client host. It will walk you through deploying a canary application to make sure Helm is properly configured.
+2. **[Deploying NuoDB using Helm Charts][5]** covers how to configure hosts to permit running NuoDB, and covers deploying your first NuoDB database using the provided Helm charts.
 
-The instructions are in two parts:
+# Getting Started with Helm 
 
-1. **[Getting Started with Helm on OpenShift][4]** covers how to install and configure Helm on a bastion host. It will walk you through deploying a canary application to make sure Helm is properly configured.
-2. **[Deploying NuoDB using Helm Charts][5]** covers how to configure hosts to permit running NuoDB, and covers deploying your first NuoDB database using the provided Helm charts.
+This section will walk you through getting both Helm client and the Tiller server up and running in your environment. If using Red Hat OpenShift, this page assumes that you already have the OpenShift `oc` client program installed locally and that you are logged into your OpenShift instance.
 
-Bear in mind there are sub-charts in subdirectories included in this distribution. Instructions provided in this specific README are more geared towards the prerequisite setup of projects, Helm and Tiller, security settings, etc. Sub-charts have details for each of the deployed components.
+There are sub-charts in subdirectories included in this distribution. Instructions provided on this page are for initial configuration of Helm and Tiller, in some cases, required security settings. Sub-charts pages include instructions for deploying each required NuoDB component.
 
-## Getting Started with Helm on OpenShift
+## Software Version Prerequisites
```

Comment:
this whole section only applies to OCP. It does not apply to vanilla K8s. It is not clear from this wording

Reply:
@vegichan  I'm not clear on what is needed here. It points to the main prereq section in the main page that covers OCP and K8s. I suggest approve the changes and make adjustments in the ENG doc PR.

Reply:
The section "Software Version Prerequisites" lists privileges. Those privileges only apply to OCP. You do not need to grant these privs in k8s.  As such, it should be clearly documented.